staticfiles/templates/project/project.html

```diff
@@ -46,18 +47,28 @@ <h1 class="md-header-darkgray">Create a New Project</h1>
                         </md-input-container>
                     </div>
                     <md-input-container class="form-input-container" flex>
-                        <label>Start Date</label>
-                        <input type="datetime" required ng-model="project.startDate">
-                        <div ng-if="project.startDate==null" ng-messages="projectForm.description.$error">
-                            <div ng-message="required">This is required.</div>
-                        </div>
+                      <label>Start Date</label>
+                      <input class='input-group date' id='datetimepicker1' required ng-model="project.startDate | date:'MM/dd/yyyy h:mm'">
+                      <div ng-if="project.startDate==null" ng-messages="projectForm.description.$error">
+                        <div ng-message="required">This is required.</div>
+                      </div>
+                      <script type="text/javascript">
+                          $(function () {
+                              $('#datetimepicker1').datetimepicker();
+                          });
+                      </script>
                     </md-input-container>
                     <md-input-container class="form-input-container" flex>
                       <label>End Date</label>
-                      <input type="datetime" required ng-model="project.endDate">
-                        <div ng-if="project.endDate==null" ng-messages="projectForm.description.$error">
-                            <div ng-message="required">This is required.</div>
-                        </div>
+                      <input class='input-group date' id='datetimepicker2' required ng-model="project.endDate | date:'MM/dd/yyyy h:mm'">
+                      <div ng-if="project.endDate==null" ng-messages="projectForm.description.$error">
+                        <div ng-message="required">This is required.</div>
+                      </div>
+                      <script type="text/javascript">
+                          $(function () {
```

Comment:
you shouldn't insert script tags into the code, please use http://mgcrea.github.io/angular-strap/#/datepickers, this should offer the required functionality in an angular centric way